specification/columns/commitmentdiscountusage.md

```diff
@@ -0,0 +1,38 @@
+# Commitment Discount Usage
```

Comment:
Some others?
- Commitment Discount Application --> Applied, Not Applied
- Commitment Discount Amortized --> Amortized, Not Amortized

Reply:
"Application" might work, but we'd have to keep the values as is. We're not tracking whether a commitment discount was applied. We're tracking whether this portion of the commitment discount was utilized or not since the unutilized portion is effectively waste and lost savings potential.

Same applies for "Amortized" and "Not Amortized". Everything that has a value would be amortized.

Reply:
More options:
- Commitment Discount Status
- Commitment Discount Utilization
- Commitment Discount Consumption

I prefer "Status".

Reply:
i will throw my vote behind status

Reply:
sorry 'Commitment Discount Status' to be exact

Reply:
I agree with Commitment Discount Status. In particular I would suggest that if in a future change adding additional values becomes an issue of discussion such as the ones Chris suggested, Status will be more future proof (and may very well be needed).

Reply:
Do we think it's necessary for 'Discount' to be in the name?

I was hoping to be able to use "Used" and "Unused" for On-Demand Capacity Reservation (ODCR) line items ([AWS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-capacity-reservations.html), [Azure](https://learn.microsoft.com/en-us/azure/virtual-machines/capacity-reservation-overview)) but ODCRs aren't typically associated with discounts so this column wouldn't make sense for that.

Reply:
Actually that suggestion was probably bad. We may want to be able to see a separate status for both Spend commitments and ODCRs since in AWS you can have an SP apply to an ODCR.

["Note that Savings Plans does not provide a capacity reservation. You can however reserve capacity with On Demand Capacity Reservations and pay lower prices on them with Savings Plans."](https://docs.aws.amazon.com/whitepapers/latest/cost-optimization-reservation-models/savings-plans.html)

So maybe you could have a line where an ODCR was "Unused" but an SP applied and was "Used"

Reply:
I'm fine with "Commitment Discount Status"

Reply:
Changed to CommitmentDiscountStatus